src/components/SeatsAnalysisViewer.vue

```diff
@@ -116,27 +128,39 @@ data() {
 setup(props) {
     let totalSeats = ref<Seat[]>([]);
     let NoshowSeats = ref<Seat[]>([]);
-    const unusedSeats = ref<Seat[]>([]);
+    const unusedSeatsInSevenDays = ref<Seat[]>([]);
+    const unusedSeatsInThirtyDays = ref<Seat[]>([]);
 
     watchEffect(() => {
         if (props.seats && Array.isArray(props.seats)) {
             totalSeats.value = props.seats;
 
+            const dateNow = new Date();
             const oneWeekAgo = new Date();
-            oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
+            const thirtyDaysAgo = new Date();
+            oneWeekAgo.setDate(dateNow.getDate() - 7);
+            thirtyDaysAgo.setDate(dateNow.getDate() - 30);
 
             NoshowSeats.value = props.seats.filter(seat => seat.last_activity_at == null);
 
-            unusedSeats.value = totalSeats.value.filter(seat => {
+            unusedSeatsInSevenDays.value = totalSeats.value.filter(seat => {
```

Comment:
this could be slightly improved if there's just one loop and we check activity date and add to counter, instead of going through all the seats twice